scene/gui/line_edit.cpp

```diff
@@ -357,8 +357,8 @@ void LineEdit::_gui_input(Ref<InputEvent> p_event) {
 			}
 		}
 
-		// Default is ENTER, KP_ENTER. Cannot use ui_accept as default includes SPACE
-		if (k->is_action("ui_text_newline", true)) {
+		// Default is ENTER. Cannot use ui_accept as default includes SPACE
+		if (k->is_action("ui_text_enter", true)) {
```

Comment:
```suggestion
		// Default is ENTER and KP_ENTER. Cannot use ui_accept as default includes SPACE
		if (k->is_action("ui_text_enter", false)) {
```

This is the change I am talking about it my comment above.

Reply:
Added in latest commit